readme.md

```diff
@@ -548,45 +554,86 @@ If the spawned process fails, [`error.stdout`](#stdout), [`error.stderr`](#stder
 
 Type: `string | Uint8Array | stream.Readable`
 
-Write some input to the `stdin` of your binary.\
-Streams are not allowed when using the synchronous methods.
+Write some input to the child process' `stdin`.\
+Streams are not allowed when using the [synchronous methods](#execasyncfile-arguments-options).
 
-If the input is a file, use the [`inputFile` option](#inputfile) instead.
+See also the [`inputFile`](#inputfile) and [`stdin`](#stdin) options.
 
 #### inputFile
 
 Type: `string`
 
-Use a file as input to the the `stdin` of your binary.
+Use a file as input to the child process' `stdin`.
 
-If the input is not a file, use the [`input` option](#input) instead.
+See also the [`input`](#input) and [`stdin`](#stdin) options.
 
 #### stdin
 
-Type: `string | number | stream.Readable | ReadableStream | undefined | URL | Iterable<string | Uint8Array> | AsyncIterable<string | Uint8Array>`\
```

Comment:
Discourage using `undefined`: `'pipe'` is clearer, and in most common cases is not needed since it is the default value.